src/FixedSizeGrid.js

```diff
@@ -65,6 +65,8 @@ const FixedSizeGrid = createGridComponent({
       default:
         if (scrollLeft >= minOffset && scrollLeft <= maxOffset) {
           return scrollLeft;
+        } else if (minOffset > maxOffset) {
+          return minOffset;
```

Comment:
I think this needs an explanatory comment, because it looks like a bug at first glance 😄

Reply:
Fair point! Now added.

Reply:
Looks like you added other places but missed this one  😆